std_msgs/Header header
LiftedPoseStamped[] poses